### PR TITLE
fix requirements, setup.py

### PR DESCRIPTION
**Pull request by wamserma**
This fixes the missing dependency on application-properties, corrects the Readme-Filename for Building on case-sensitive platforms and puts the resource file in the correct location to fix usage in virtual envs.

Please consider releasing 0.8.1 including this fix.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -80,10 +80,5 @@ def ensure_scripts(linux_scripts):
     classifiers=PROJECT_CLASSIFIERS,
     scripts=PACKAGE_SCRIPTS,
     packages=PACKAGE_MODULES,
-    data_files=[
-        (
-            "Lib/site-packages/pymarkdown/resources",
-            ["pymarkdown/resources/entities.json"],
-        )
-    ],
+    include_package_data=True,
```
Comment (jackdewinter) [inline setup.py:83]: Thanks for pointing these out.  Can you explain to me (any links would be helpful) the problem that this final change solves?

Comment (wamserma) [reply setup.py:83]: Well, it is in the official documentation (sort of, [setuptools 57.4.0](https://setuptools.readthedocs.io/en/latest/userguide/datafiles.html))
Basically, `data_files` installs files relative to the used interpreter, while `include_data_files` takes files listed in the manifest and installs them relative to the package, which is more robust and the preferred way,

**Comment by wamserma** [comment]
@jackdewinter Is there anything blocking this from being merged?

**Comment by jackdewinter** [comment]
Sorry about the delay, have been extremely busy.

**Comment by wamserma** [comment]
Could you release a 0.8.1 (or maybe 0.9.0) with these changes?

**Pull request merged**